parser/src/layout.rs

```diff
@@ -173,6 +173,7 @@ where
         loop {
             // Retrieve the current indentation level if one exists
             let offside = match (&token.value, self.indent_levels.last().cloned()) {
+                (&Token::ShebangLine(_), _) => return Ok(token),
```

Comment:
No worries, now that I know it shouldn't be a big deal in the future.  The layered approach is definitely the way to go overall it just had a bit of a learning curve.  This was my first rust code and my first time using lalrpop so I figured I was missing something.